app/ante/authz_test.go

```diff
@@ -274,7 +274,7 @@ func (suite *AnteTestSuite) TestRejectDeliverMsgsInAuthz() {
 					},
 				),
 			},
-			expectedCode: sdkerrors.ErrUnauthorized.ABCICode(),
+			expectedCode: sdkerrors.ErrUnpackAny.ABCICode(),
```

Comment:
err code change, for [more](https://github.com/cosmos/cosmos-sdk/pull/12184/files) info